ENIGMAsystem/SHELL/Platforms/General/PFwindow.cpp

```diff
@@ -329,4 +329,8 @@ void window_set_region_size(int w, int h, bool adaptwindow) {
   enigma::compute_window_size();
 }
 
+bool window_has_focus() {
+  return enigma::game_window_focused;
```

Comment:
@RobertBColton Is this what you meant?

Reply:
@RobertBColton also, my hymen broke, could you please check out my execute_shell_for_output pr?